task_8.py

```diff
@@ -0,0 +1,13 @@
+# Задача 8: Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, 
+# если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+# *Пример:*
+# 3 2 4 -> yes
+# 3 2 1 -> no
+
+n = int(input())
+m = int(input())
+k = int(input())   
+if k < m*n and k != m*n and (k%m == 0 or k%n == 0):
```

Comment:
Второе условие не нужно, так как первое его исключает. Но первое условие можно расширить по нижней границе допустимых значений.
```suggestion
if k < m * n and (k % m == 0 or k % n == 0):
```